app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.map do |movie|
+      movie_params(movie)
+    end
+    render json: @movies
+  end
+
+  def show
+    movie = Movie.find(params[:id])
+    @transformed_movie = movie_params(movie)
+    render json: @transformed_movie
```

Comment:
You don't need to build a response hash with the `movie_params` method. If you want to respond with JSON that contain only movie attributes, you can do it with: 
```render json: movie```

And you can do the same in the `index` action:
```
@movies = Movie.all
render json: @movies
```